mobile/src/main/java/ie/yesequality/yesequality/NotificationActivity.java

```diff
@@ -0,0 +1,112 @@
+package ie.yesequality.yesequality;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.Switch;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+
+
+public class NotificationActivity extends Activity {
+
+    @InjectView(R.id.switchDayBefore) Switch dayBefore;
+    @InjectView(R.id.switchOnDay) Switch onDay;
+    @InjectView(R.id.okButton) Button okButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_notification);
+        ButterKnife.inject(this);
+
+        okButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
You can use butterknife for the clicks too. @OnClick(R.id.okButton) void onOkClick() { // do something }